index.php

```diff
@@ -20,14 +20,44 @@
     <div class="flex"><h1>ODD SOCKS COLLECTION</h1></div>
     <div class="flex"><h3>a place to keep a collection of your odd socks</h3></div>
 </div>
-<div class="container flex">
-    <form method="get">
-    </form>
-    <div class="collection flex">
-        <?php
-        echo displaySocksCollection(connectDB());
-        ?>
+<div class="interactions flex">
+    <div class="interaction add flex">
+        <h1>ADD</h1>
+        <form method="get" class="flex">
+            <div class="small-inputs flex">
+                <label for="name">Name:</label>
+                <input name="name" id="name" type="text" required="required">
+                <label for="size">Size:</label>
+                <?php
+                echo createDropdown('size');
+                ?>
+                <label for="pattern">Pattern:</label>
+                <?php
+                echo createDropdown('pattern');
+                ?>
+                <label for="color">Color:</label>
+                <?php
+                echo createDropdown('color');
+                ?>
+            </div>
+            <div class="description-input flex">
+                <label for="description">Description:</label>
+                <textarea name="description" id="description"></textarea>
+                <input class="submit" type="submit">
+            </div>
+        </form>
     </div>
+    <div class="interaction">
+        <h1>EDIT</h1>
+    </div>
+    <div class="interaction">
+        <h1>DELETE</h1>
+    </div>
```

Comment:
EDIT and DELETE are not part of this story, remove these.

We don't do work that isn't in the story - what if we don't get DELETE done and a big box is there on the page where it "will go", what will our client or users think? What awkward questions are they going to ask? Are they gong to accept the software with big obvious holes in it?

Reply:
will do